src/Plugin/Action/StrawberryfieldJsonPatch.php

```diff
@@ -242,6 +241,7 @@ public function execute($entity = NULL) {
                 );
               };
               if ($this->configuration['simulate']) {
+                $this->messenger()->addMessage('In simulation Mode');
```

Comment:
Hey, thinking about this twice (sorry, slow), better just remove this. Because on 1000 objects this is basically 1000 messages right? Or maybe it can be send before the execute is send? like on page 1999? What do you think?

Reply:
Ah ok, that makes sense. Will just remove for now.